app/src/main/java/http_server/App.java

```diff
@@ -9,8 +9,8 @@ public class App {
     public static void main(final String[] args) throws IOException {
         Console console = new Console();
 
-        RequestHandler requestHandler = new RequestHandler();
-        Router router = new Router(requestHandler);
+        Router router = new Router();
+        ResponseHandler responseHandler = new ResponseHandler();
       
```

Comment:
I made the decision to remove the RequestHandler because I found its functionality to be somewhat redundant, excessively complex, and overall difficult to understand. This became particularly evident when the Response was returned as an object instead of a string. The code became more dynamic and simplified within the RequestHandler to the extent that it felt like I was simply passing an object through two methods (first in RequestHandler and then again in Router). However, if you believe that removing it was a mistake on my part, I am certainly open to reverting the change. Perhaps the decision to remove it was made too flippantly.